articles/errors/libraries/auth0-js/invalid-token.md

```diff
@@ -0,0 +1,23 @@
+---
+public: false
+---
+# HS256-Signed Tokens
+
+Error: calls made to the **/userInfo** return the following:
```

Comment:
Tokens signed with the HS256 algorithm cannot be properly validated. Instead, starting from auth0.js@v9 and lock@v11, a call to `/userinfo` is made with a valid `access_token`. If you don't have an access_token, you'll see the error: